docs/upgrade-guide.md

```diff
@@ -22,6 +22,21 @@
   + `pixelsPerMeter` -> `unitsPerMeter`
   + `metersPerPixel` -> `metersPerUnit`
 
+##### Multi-view state handling
+
+We have fixed a bug when using `initialViewState` with multiple views. In the past, the state change in one view is unintendedly propagated to all views. As a result of this fix, multiple views (e.g. mini map) are no longer synchronized by default. To synchronize them, define the views with an explicit `viewState.id`:
+
```

Comment:
How to sync different views? Is there an example? Or maybe briefly mention here

Reply:
There's a code sample right after this paragraph.

Reply:
I saw that, do you mean, to make different view synchronized, the user doesn't need do anything other than providing an id for each view?

Reply:
A view can contain a `viewState` prop which has a different id from the view itself.